client/app/dedicatedCloud/components/stepper/controller.js

```diff
@@ -0,0 +1,153 @@
+import _ from 'lodash';
+
+import Step from './step';
+
+export default class Stepper {
```

Comment:
Could it make sense to have a `oui-progress-tracker` component to handle this or is it too specific to your needs ?